### PR TITLE
Updated license things for jadtechwiki

### PR DESCRIPTION
**Pull request by PlasmaLight**

### CODE REVIEW

**Comment by PlasmaLight** [comment]
Closed it because I thought that something was wrong, but it seems that it is fine so I will repoen it.

File: LocalSettings.php [hunk 1]
```diff
@@ -2155,6 +2155,7 @@
 	'wgRightsIcon' => array(
 		'default' => 'https://meta.miraheze.org/w/resources/assets/licenses/cc-by-sa.png',
 		'incubatorwiki' => 'https://meta.miraheze.org/w/resources/assets/licenses/cc-by-sa.png',
+		'jadtechwiki' => //static.miraheze.org/jadtechwiki/d/d8/CopyrightIcon.png',
```
Comment (Reception123) [inline LocalSettings.php]: It's done like that in ManageWiki, but here you must use  "//$wmgUploadHostname instead of static.mh

**Comment by PlasmaLight** [comment]
I have changed what you asked me to change.

File: LocalSettings.php [hunk 2]
```diff
@@ -2155,6 +2155,7 @@
 	'wgRightsIcon' => array(
 		'default' => 'https://meta.miraheze.org/w/resources/assets/licenses/cc-by-sa.png',
 		'incubatorwiki' => 'https://meta.miraheze.org/w/resources/assets/licenses/cc-by-sa.png',
+		'jadtechwiki' => //$wmgUploadHostname/jadtechwiki/d/d8/CopyrightIcon.png',
```
Comment (Reception123) [inline LocalSettings.php]: Don't forget the ' at the beginning, or else the PHP code will fail

**Pull request merged**